pyproject.toml

```diff
@@ -35,6 +35,7 @@ pyinstaller = { version = "^4.1", optional = true }
 
 [tool.poetry.dev-dependencies]
 pyqt5 = "^5.15.0"
+pyqt5-sip = "^12.8.1"
```

Comment:
Are you sure this should be a dev dependency? You import it [here](https://github.com/aotuai/brainframe-qt/pull/120/files#r614275966)

Reply:
I only made it a dev dependency because pyqt5 itself is a dev dependency.